spec/i18n_backend_spec.rb

```diff
@@ -45,4 +45,19 @@
     I18n.locale = en_locale
     expect(I18n.t(key)).to eq en_value
   end
+
+  it '#pseudo mode' do
+    Sgtn.pseudo_mode = true
```

Comment:
When do integration project, can the product team's developer enable the pseudo feature on the product's backend by this codeline 'Sgtn.pseudo_mode = true' as well?

Reply:
I just want to confirm the usage.

Reply:
Yes.